tests/functional/spec/household-relationship.spec.js

```diff
@@ -25,23 +24,45 @@ describe('Household relationship', function() {
 
   it('Given I answer the relationship questions for Joe Bloggs when answer the relationship questions for Jane Doe then I should not have to enter relationship details for John Doe', function() {
     // Given
-    openQuestionnaire(schema)
+    openQuestionnaire('test_relationship_household.json')
     HouseholdCompositionPage.setPersonName(0, 'Joe Bloggs')
       .addPerson()
       .setPersonName(1, 'Jane Doe')
       .addPerson()
       .setPersonName(2, 'John Doe')
       .submit()
-    HouseholdRelationshipPage.clickWhoIsRelatedHusbandOrWife(0)
-      .clickWhoIsRelatedSonOrDaughter(1)
+    HouseholdRelationshipPage.setWhoIsRelatedHusbandOrWife(0)
+      .setWhoIsRelatedSonOrDaughter(1)
       .submit()
 
     // When
-    HouseholdRelationshipPage.clickWhoIsRelatedHusbandOrWife(0)
+    HouseholdRelationshipPage.setWhoIsRelatedHusbandOrWife(0)
       .submit()
 
     // Then
     expect(SummaryPage.isOpen()).to.be.true
   })
 
+  it('Given I answer the relationship questions for Joe Bloggs when I click previous on the Jane Doe relationship question then the relationships for John Bloggs should still be selected', function() {
+    // Given
+    openQuestionnaire('test_relationship_household.json')
+    HouseholdCompositionPage.setPersonName(0, 'Joe Bloggs')
+      .addPerson()
+      .setPersonName(1, 'Jane Doe')
+      .addPerson()
+      .setPersonName(2, 'John Doe')
+      .submit()
+    HouseholdRelationshipPage.setWhoIsRelatedHusbandOrWife(0)
+      .setWhoIsRelatedSonOrDaughter(1)
+      .submit()
+
+    // When
+    browser.back()
```

Comment:
Is this intended to actually invoke the browser's back button? Do we need another test for clicking the previous link?